Libplanet/Blocks/Block.cs

```diff
@@ -431,7 +426,9 @@ internal BlockHeader GetBlockHeader()
             // FIXME: When hash is not assigned, should throw an exception.
             return new BlockHeader(
                 index: Index,
-                timestamp: Timestamp.ToString(TimestampFormat, CultureInfo.InvariantCulture),
+#pragma warning disable MEN002 // line is too long
+                timestamp: Timestamp.ToString(BlockHeader.TimestampFormat, CultureInfo.InvariantCulture),
```

Comment:
How about using local variables instead of disabling the warning?

Reply:
🆗 I'll fix it.